edx_django_utils/plugins/plugin_manager.py

```diff
@@ -0,0 +1,54 @@
+"""
+Adds support for first class plugins that can be added to the edX platform.
```

Comment:
Search for "edX Platform" and "edx-paltform" throughout this PR and update to "an IDA" as necessary.

Reply:
I would like to leave it as is for now, especially cause I think we need to do some more work to make the plugins system work for other IDAs.

Reply:
I think your comment is out-of-date.  Is that true?

Reply:
I think so, I will see what needs to be changed